onadata/apps/api/tasks.py

```diff
@@ -65,6 +68,14 @@ def delete_xform_async(xform_id):
     xform.soft_delete()
 
 
+@periodic_task(
+    run_every=(crontab(0, 0, day_of_month='15')))
+def delete_user_async(username):
```

Comment:
I think that based on our conversation this should be deleting inactive users.  I think you should consider doing:

1. This function doe snot need to accept any arguments.  You just need to query for inactive users within the function/task, and then delete them.
2. I think the established "style" of defining periodic tasks is in settings using `CELERY_ROUTES` e.g. https://github.com/onaio/playbooks/blob/cff9d8230a37381ec478d11412232c98a8e59ff2/templates/opt/onadata/settings/local_settings.py#L256 so you might not need the `@periodic_task` decorator